libs/video-view/src/components/settings-drawer/index.tsx

```diff
@@ -0,0 +1,182 @@
+import { Box, Flex, Heading, Slide, Stack } from '@chakra-ui/react';
```

Comment:
Just found out the settings drawer is in the video view. We should bring the settings component out of the video view, they should not be bundled together.

Reply:
Per our discussion, I have changed the implementation of the settings drawer such that it is handled at the application level